File: java/URI_1018.java
package URI;
import java.util.Scanner;

public class URI_1018 {

	public static void main(String[] args) {		 
		 Scanner sc = new Scanner(System.in);
		 
		 int N, qtde;
		 int[]cedulas = new int[] {100, 50, 20, 10, 5, 2, 1};
		 
		 N = sc.nextInt();
		 
		 System.out.println(N);			 
		 for(int notas : cedulas) {			
			 qtde = N / notas;
			 N = N % notas;			 
			 System.out.printf("%d nota(s) de R$ %d,00%n", qtde, notas);
		 }			 				 
		 sc.close(); 		 
	}

}

File: kotlin/URI_1018.kt
import java.util.*

val notas = intArrayOf(100,50,20,10,5,2,1)

fun main(args: Array<String>) {
    val sc = Scanner(System.`in`)
    var reais = sc.nextInt()
    println(reais)
    for(nota in notas) {
        val quantidade = reais/nota
        println("$quantidade nota(s) de R$ $nota,00")
        reais -= quantidade * nota
    }
}